definitions/npm/marked-man_v0.1.x/flow_v0.28.x-/marked-man_v0.1.x.js

```diff
@@ -0,0 +1,64 @@
+import marked from 'marked'
```

Comment:
Wouldn't it be better to use the `$` convention? So exporting `marked$Marked` from the marked libdef?